uswds/uswds-sandbox
# USWDS-Sandbox - Zombie test form: Replace form links with buttons

# Summary

Replace next step links with actual buttons. 

Change made a result of our Usability testing [findings](https://github.com/uswds/uswds/issues/5527).

## Related issue

Closes #113

## Related pull requests

> **Warning**
Work should be merged after #103
> 

## Preview link

[Zombie test form →](https://federalist-d5e7c07c-6ffa-4b8e-935f-49a7df24a505.sites.pages.cloud.gov/preview/uswds/uswds-sandbox/cm-zombie-defense-update-buttons/)

## Problem statement

Buttons within forms should be actual buttons and not links styled as buttons as stated by our [button guidance](https://designsystem.digital.gov/components/button/).

## Solution

Replace all form links styled as buttons with actual buttons. 

Buttons navigate to new pages via `onclick="location.href = '/page-destination'`

## Testing and review

1. Navigate through test form with and without screen reader.
2. Confirm the next and previous page buttons work as expected.
3. Confirm screen reader callouts are accurate.
4. Confirm screen readers are able to target the buttons using button shortcuts.
    - There doesn’t appear to be a shortcut for VoiceOver
    - For JAWs and NVDA this is `b` **********(forwards)********** or `shift + b` ***********(backwards)***********

### Testing checklist

- [ ]  Form navigates as expected
- [ ]  No visual regression
- [ ]  Screen readers call out buttons as expected
- [ ]  Screen reader navigation works as expected
- [ ]  Confirm this is the proper method to navigate through a form with buttons
